### DISCUSSION TITLE
Where is the Android version, actually?

### DISCUSSION THREAD

**Discussion by rugk** [ppy/osu#18296] [Q&A] [answered]
I hear people in the issues talk about Android, but where is it…?

Neither on https://osu.ppy.sh/home/download (only Windows and macOS links/mentioned there; Linux [can be retrived from Flathub](https://flathub.org/apps/details/sh.ppy.osu)) I can find it, …
![grafik](https://user-images.githubusercontent.com/11966684/168496858-dec3a5dc-ebd1-49ef-8748-412f28611a01.png)

… nor [on Google Play](https://play.google.com/store/search?q=osu!&c=apps) (searching for “osu!”):
![grafik](https://user-images.githubusercontent.com/11966684/168496830-ec272ada-2854-4849-a057-718c2e0b753c.png)

(At least not the official version/client.) 

## Question

So where can I download the Android client?

**Comment by zzahkaboom24** [comment, marked as answer]
https://github.com/ppy/osu

Scroll down to "Running osu!", you will find android there otherwise click here for the latest version as of now: https://github.com/ppy/osu/releases/download/2022.515.0/sh.ppy.osulazer.apk

**Reply by rugk** [reply]
But… not yet released in any app store? :eyes:

**Reply by rugk** [reply]
Direct link to that section BTW: https://github.com/ppy/osu#running-osu

And direct link to the _latest_ version: https://github.com/ppy/osu/releases/latest/download/sh.ppy.osulazer.apk

**Reply by zzahkaboom24** [reply]
> But… not yet released in any app store? :eyes: 

no, and it probably wont happen that soon yet (?), first they focus on the desktop version of the game, then the mobile one
